internal/distro/rhel9/imagetype.go

```diff
@@ -251,6 +236,9 @@ func (t *imageType) Manifest(bp *blueprint.Blueprint,
 			cw.DisabledServices = services.Disabled
 		}
 		w = cw
+	} else {
+		// workload packages should use the payload repositories
+		w.SetRepos(payloadRepos)
```

Comment:
I'm not completely sure about this 🤔 For internal workloads like SAP or EAP, I would argue that we don't want to use user-specified repos for depsolving the package set. I think that we will still need to handle Workloads in two ways (and even when merging later). One situation is any internal workload that should be treated as the base image (WRT package set and used repos) and another situation is a custom workload based on user input.

I think that if we would apply customizations (read as workloads) in a separate pipeline per workload, that could remove the problem of merging workloads and would make the separation easier (IMO). I'm not suggesting that you should do it now, but I think that we should keep that in mind for the future...

Reply:
> For internal workloads like SAP or EAP, I would argue that we don't want to use user-specified repos for depsolving the package set.

I'm inclined to agree but not sure if we would need to support it eventually, maybe for testing or for internal builds.  I think I'm going to remove it for now and keep support for payload repos only on custom workloads.  We can revisit it later if a requirement comes around.

Reply:
I've removed this and adjusted tests accordingly.  The EAP image type, which is currently the only one that specifies a workload, drops any payload repos now.